[CON-1406] feat: Added snapchatads write functionality (#1841)
amp-labs/connectors: providers/snapchatads/README.md

Below objects having url like v1/{organization_id}/objectName

-------------------------------------------------------------------
| Object                  | Resource               | Method       |
| ----------------------- | ---------------------- | -------------|
| fundingsources          | fundingsources         | read         |
| billingcenters          | billingcenters         | read, write  |
| transactions            | transactions           | read         |
| adaccounts              | adaccounts             | read, write  |
| members                 | members                | read, write  |
| roles                   | roles                  | read, write  |
-------------------------------------------------------------------
 
Notes:
- The organization_id is retrieved using the postAuthentication method.
- Currently, we only support objects that include the organization_id in the URL path.

> - The API documentation specify using the POST method to update billingcenters, it turns out the PUT method works just as well.

Reviewer: ```suggestion
- The API documentation mentions that we must use the POST method to update billingcenters, but it turns out that the PUT method works too.
```
